openedx/adg/lms/applications/tests/conftest.py

```diff
@@ -49,3 +52,24 @@ def education_inline():
 @pytest.fixture
 def work_experience_inline():
     return WorkExperienceInline(UserApplication, adg_admin_site)
+
+@pytest.fixture
+def user_applications_with_different_business_lines():
+    UserApplicationFactory(business_line=BusinessLineFactory(title=TITLE_BUSINESS_LINE_1))
+    UserApplicationFactory(business_line=BusinessLineFactory(title=TITLE_BUSINESS_LINE_2))
```

Comment:
It can be written in a more concise way
```suggestion
    UserApplicationFactory(business_line__title=TITLE_BUSINESS_LINE_1)
    UserApplicationFactory(business_line__title=TITLE_BUSINESS_LINE_2)
```